project/core/templates/global_nav.html

```diff
@@ -1,4 +1,5 @@
-
+{% load static %}
+{% load notifications_tags %}
```

Comment:
Do we need to load notification_tags here or is “load static” all that is needed?

Reply:
Hi Brylie, thanks for responding.

Sorry I should have mentioned this in my pull request, when I don't load in notifications_tags then I get a template syntax error for live_notify_badge tag.
![image](https://user-images.githubusercontent.com/46718979/233779679-da38d584-de28-47f2-80fb-1e43f355eefe.png)